TrustWalletCore.podspec

```diff
@@ -18,129 +18,103 @@ Pod::Spec.new do |s|
+    "#{protobuf_source_dir}/src/google/protobuf/stubs/bytestream.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/stubs/common.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/stubs/int128.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/stubs/status.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/stubs/statusor.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/stubs/stringpiece.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/stubs/stringprintf.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/stubs/structurally_valid.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/stubs/strutil.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/stubs/substitute.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/stubs/time.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/text_format.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/timestamp.pb.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/type.pb.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/unknown_field_set.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/util/delimited_message_util.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/util/field_comparator.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/util/field_mask_util.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/util/internal/datapiece.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/util/internal/default_value_objectwriter.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/util/internal/error_listener.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/util/internal/field_mask_utility.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/util/internal/json_escaping.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/util/internal/json_objectwriter.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/util/internal/json_stream_parser.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/util/internal/object_writer.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/util/internal/proto_writer.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/util/internal/protostream_objectsource.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/util/internal/protostream_objectwriter.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/util/internal/type_info.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/util/internal/utility.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/util/json_util.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/util/message_differencer.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/util/time_util.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/util/type_resolver_util.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/wire_format.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/wire_format_lite.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/wrappers.pb.cc",
   s.exclude_files =
     'trezor-crypto/src/rand.c',
-    'trezor-crypto/include/TrezorCrypto/**/base58.h'
```

Comment:
exclude   'trezor-crypto/include/TrezorCrypto/**/base58.h' ?

Reply:
preserve_paths and exclude_files can't work at the same time, this is reverted old settings, I found another workaround to fix this

Reply:
see https://github.com/trustwallet/ios/commit/1f1e14060dbee7f8108ab972b41317993ac4f1ca